test/WebJobs.Script.Tests.Integration/TestScripts/Node/HttpTrigger/function.json

```diff
@@ -4,8 +4,7 @@
             "type": "httpTrigger",
             "name": "request",
             "direction": "in",
-            "methods": [ "get", "post" ],
-            "authLevel": "anonymous"
```

Comment:
Moving these tests back to using a key - I think someone was lazy and just made these anonymous during the v2 port

Reply:
That's not correct. The port maintained the same behavior as V1. A quick verification shows when, and why, those changes were made.

Reply:
Original test used default auth by intention, but when it was moved it was made anonymous, because the test suite apparently hadn't been updated yet to properly send the key. I'm saying we should have kept the key auth so some portion of the e2e tests continue to test the full auth pipeline.